crates/iota-core/src/checkpoints/checkpoint_executor/tests.rs

```diff
@@ -56,37 +56,47 @@ pub async fn test_checkpoint_executor_crash_recovery() {
     let checkpoints = sync_new_checkpoints(
         &checkpoint_store,
         &checkpoint_sender,
-        2 * buffer_size,
+        buffer_size,
```

Comment:
what was the reason to change the number of synced checkpoints everywhere?

Reply:
To decrease the number of checkpoints to sync.